src/detect-engine-threshold.c

```diff
@@ -281,6 +319,99 @@ static inline void RateFilterSetAction(Packet *p, PacketAlert *pa, uint8_t new_a
     }
 }
 
+static int IsThresholdQualify(DetectThresholdEntry* lookup_tsh, const DetectThresholdData *td, uint32_t packet_time)
```

Comment:
this function name doesn't really help me understand what it does. Can you rename and/or add a comment explaining the purpose?